packages/applications/projectors/src/subscribers/période/index.ts

```diff
@@ -12,8 +12,8 @@ export type SubscriptionEvent = (Période.PériodeNotifiéeEvent & Event) | Rebu
 export type Execute = Message<'System.Projector.Periode', SubscriptionEvent>;
 
 export const register = () => {
-  const handler: MessageHandler<Execute> = async (event) => {
-    await match(event)
+  const handler: MessageHandler<Execute> = (event) => {
```

Comment:
Tu peux même simplifier du coup

```suggestion
  const handler: MessageHandler<Execute> = (event) => match
```

Reply:
C'est vrai qu'au début je voulais être explicite pour éviter les erreurs mais avec la règle eslint l'erreur n'est plus possible.
J'ai mis à jour tous les projecteurs du coup :pray: